server/routerlicious/packages/routerlicious-base/src/utils/sessionHelper.ts

```diff
@@ -76,6 +76,7 @@ export async function getSession(ordererUrl: string,
 
     // The tempSession.isSessionAlive would be updated as whether the session was alive before the request came.
     tempSession.isSessionAlive = isSessionAlive;
-    Lumberjack.info(`Returning the session: ${JSON.stringify(tempSession)}`, lumberjackProperties);
+    Lumberjack.info(`Returning the session with enableDiscovery flag as true: ${JSON.stringify(tempSession)}`,
```

Comment:
I don't think we need to specify the enable discover flag is true here since it this method will be called only if it is true

Reply:
Got it. Thanks. Removed this flag. :)